src/lib/Renderer.ts

```diff
@@ -353,7 +343,127 @@ class Renderer {
       return { error: 'unsafe_redirect' };
     }
 
-    return { statusCode, headers, body, timeout, resolvedUrl };
+    return { statusCode, headers, body, resolvedUrl };
+  }
+
+  private async _processLogin(task: TaskParams): Promise<TaskResult> {
+    /* Setup */
+    const { url } = task;
+    const { context, page } = await this._newPageWithContext(task);
+
+    try {
+      await this._goto(page, url);
+    } catch (e) {
+      return { error: e.message, timeout: Boolean(e.timeout) };
+    }
+
+    const textInput = await page.$('input[type=text], input[type=email]');
+    if (!textInput) {
+      return { error: `field_not_found: input[type=text], input[type=email]` };
+    }
+    await textInput!.type(task.login!.username);
+
+    let passwordInput = await page.$('input[type=password]');
```

Comment:
that means you are not handling the 3 steps login no? (algolia)
domain page + login page + password page

Reply:
Yeah this is for username/password login only.